### PR TITLE
HOTT-2196 Fix for no end date on quota definitions

### PR DESCRIPTION
**Pull request by jebw**
### Jira link

HOTT-2196

### What?

I have added/removed/altered:

- [x] Allow for missing end date in `govuk_date_range` helper

### Why?

I am doing this because:

- It is used on the quota definitions popup, and there are quotas without end dates

### Have you? (optional)

- [ ] Reviewed view changes with stake holders
- [x] Validated mobile responsive behaviour of view changes

### Deployment risks (optional)

- Low, increases flexibility of date range helper and fixes an existing issue

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -130,7 +130,11 @@ def present_from_to(from, to)
   end
 
   def govuk_date_range(from, to)
-    "#{from.to_date.to_formatted_s(:long)} to #{to.to_date.to_formatted_s(:long)}"
+    if to.present?
+      "#{from.to_date.to_formatted_s(:long)} to #{to.to_date.to_formatted_s(:long)}"
```
Comment (alexdesi) [inline app/helpers/application_helper.rb:134]: just to check: do we need "From " at the start of this?

File: spec/helpers/application_helper_spec.rb [hunk 2]
```diff
@@ -342,6 +342,18 @@
 
       it { is_expected.to eql '1 January 2022 to 1 June 2022' }
     end
+
+    context 'with blank :to date' do
```
Comment (alexdesi) [inline spec/helpers/application_helper_spec.rb]: nitpicking:
Does 
`context 'when :to date is blank' do`
read better? 🤔

File: spec/helpers/application_helper_spec.rb [hunk 3]
```diff
@@ -342,6 +342,18 @@
...
+      subject { govuk_date_range from, '' }
+
+      it { is_expected.to eql 'From 1 January 2022' }
+    end
+
+    context 'with nil :to date' do
```
Comment (alexdesi) [inline spec/helpers/application_helper_spec.rb]: same as before

**Review by alexdesi — APPROVED** [review]
It looks good ✅ 
Just a couple of nitpickings

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (jebw) [reply app/helpers/application_helper.rb:134]: Not for our current usage though appreciate that conflicts with how GovUK guidelines say we should be formatting dates

**Pull request merged**